topologies/advanced.json

```diff
@@ -1,37 +1,37 @@
 {
-    "enableExplorer": false,
-    "nodes": [
+  "enableExplorer": false,
```

Comment:
`firstNode` and `secondNode` had to be renamed here in order for the `verifyEnv` function to be able to work with both `basic.json` as well as `advanced.json` topologies)

(see issue [RLP-1084](https://jirainfuy.atlassian.net/browse/RLP-1084))